src/App.js

```diff
@@ -1,10 +1,172 @@
+                labelText = "Voornaam"
+            />
+            <Input
+                idAttribute = "lastname-field"
+                nameAttribute = "lastname"
+                inputType = "text"
+                stateValue = {lastname}
+                stateSetter = {setLastname}
+                labelText = "Achternaam"
+            />
+            <Input
+                idAttribute = "age-field"
+                nameAttribute = "age"
+                inputType = "number"
+                stateValue = {age}
+                stateSetter = {setAge}
+                labelText = "Leeftijd"
+            />
+            <Input
+                idAttribute = "post-field"
+                nameAttribute = "post"
+                inputType = "text"
+                stateValue = {post}
+                stateSetter = {setPost}
+                labelText = "Postcode"
+            />
+            <p>Bezorgfrequentie</p>
+            <select
+            name = "delivery"
+            id = "delivery-field"
+            value = {deliveryFrequency}
+            onChange = {(e) => setDeliveryFrequency(e.target.value)}>
+                <option value = "week">iedere week</option>
+                <option value = "every-other-week">om de week</option>
+                <option value = "month">iedere maand</option>
+            </select>
+           <div>
+               <input
+                   type = "radio"
+                   id = "choice-day"
+                   name = "delivery-time"
+                   value = "daytime"
+                   checked = {deliveryTime === "daytime"}
+                   onChange = {(e) => setDeliveryTime(e.target.value)}/>
+               <label htmlFor = "choice-day">Overdag</label>
+
+               <input
+                   type = "radio"
+                   id = "choice-evening"
+                   name = "delivery-time"
+                   value = "in-the-evening"
+                   checked = {deliveryTime === "in-the-evening" }
+                   onChange = {(e) => setDeliveryTime(e.target.value)}
+               />
+               <label htmlFor = "choice-evening">'s Avonds</label>
+           </div>
+
+            <label htmlFor = "comments-field">Opmerkingen</label>
+            <textarea
+                name = "comments"
+                id = "comments-field"
+                value = {comments}
+                onChange = {(e) => setComments(e.target.value)}
+                rows={10}
+                cols={50}
+            />
+                <input
+                    type = "checkbox"
+                    name = "agree"
+                    id = "agree-field"
+                    value = {agree}
+                    onChange = {(e) => setAgree(e.target.checked)}
+                />
+                <label htmlFor = "agree-field">Ik ga akkoord met de voorwaarden</label>
+
+            <button type = "submit" className = "reset-submit" onClick = {() => resetAll()}>Verzend</button>
```

Comment:
Het is net als wiskunde waarbij je wat er links en rechts van de => staat kan wegstrepen 😉
```suggestion
            <button type = "submit" className = "reset-submit" onClick = {resetAll}>Verzend</button>
```